azurekvp/src/lib.rs

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+pub mod main;
```

Comment:
Why is that needed?
It feels like some unnecessary line that can be cleaned up.

Reply:
I think I had originally set it up this way before I was sure how we were going to split the tracing & main functionality (and forgot to remove it later on) but you are right, we don't need it!! I will remove it!